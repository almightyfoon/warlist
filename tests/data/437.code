c5501b_-0h0z1ggj0u9v3y3y3y4L0w3M3i3i___-0A0G0w0w0w3y3y3y3W3-f3f33m3o3m3ohP