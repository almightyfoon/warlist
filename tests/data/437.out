https://conflictchamber.com/#c5501b_-0hk-1ggjmv9v3y3y3y4L0w3Mmgmg___-0A0G0w0w0w3y3y3y3W3-f3f33m3o3m3ohP

Retribution Army - 70 / 75 points
[Theme] Forges of War

[Rahn 1] Adeptis Rahn [+26]
 - Chimera [8]
 - Helios [34]
 - Phoenix [17]
 - Sylys Wyshnalyrr, the Seeker [0(4)]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
House Shyeel Artificer [0(5)]
 - Hydra [15]
House Shyeel Magister [0(4)]
House Shyeel Battle Mages [8]
House Shyeel Battle Mages [8]

Retribution Army - 75 / 75 points
[Theme] Shadows of the Retribution

[Kaelyssa 1] Kaelyssa, the Night's Whisper [+29]
 - Hydra [15]
 - Hydra [15]
 - Hydra [15]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Arcanist Mechanik [2]
Eiryss, Mage Hunter of Ios [0(7)]
Narn, Mage Hunter of Ios [0(6)]
Soulless Voidtracer [2]
Soulless Voidtracer [2]
Mage Hunter Strike Force (max) [16]
 - Mage Hunter Strike Force Commander [4]
Mage Hunter Strike Force (max) [16]
 - Mage Hunter Strike Force Commander [4]
Spears of Scyrah [9]
